### PR TITLE
Document the JSON API 

### PR DESCRIPTION
**Pull request by brendandeere**
Document the JSON API endpoints for creating and updating subscritpion line items

### CODE REVIEW

File: docs/api/v1/subscription_line_items.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Susbcription Line Items
```
Comment (alexblackie) [inline docs/api/v1/subscription_line_items.md]: Subscription

File: docs/api/v1/subscription_line_items.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+SolidusSubscritpions::LineItems are analogous to Spree::LineItem. They indicate
```
Comment (alexblackie) [inline docs/api/v1/subscription_line_items.md]: SolidusSubscriptions

File: docs/api/v1/subscription_line_items.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+the details of a subscription prior to a purchase being made.
+
+## PATCH `/api/v1/line_item/:id`
+*Authentication Required*
+
+Update the details of a specific SolidusSubscriptions::LineItem. i.e. From the
```
Comment (alexblackie) [inline docs/api/v1/subscription_line_items.md]: `i.e.` should be an `e.g.`

File: docs/api/v1/subscription_line_items.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+cart page alongside Spree::LineItems
```
Comment (alexblackie) [inline docs/api/v1/subscription_line_items.md]: missing punctuation

File: docs/api/v1/subscription_line_items.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+### Example params:
+```js
+{
+  "id" => 1,
+  "token" => 'abc123', // Spree api token
```
Comment (alexblackie) [inline docs/api/v1/subscription_line_items.md]: ahhh mixing quotes

File: docs/api/v1/subscription_line_items.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+This endpoint accepts the default subscription_line_item_attributes (which are
+configurable) except for the :subscribable_id. These atrributes are:
+- `:quantity`
+- `:interval`
+- `:max_installments`
+
+### Example response:
+{
```
Comment (alexblackie) [inline docs/api/v1/subscription_line_items.md]: needs to be fenced in code block

File: docs/api/v1/subscription_line_items.md (hunk 5 above)
Comment (brendandeere) [inline docs/api/v1/subscription_line_items.md]: search and replace fail

File: docs/api/v1/orders.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+# Spree::Orders
+Creating SolidusSubscriptions::LineItems though the Solidus API. The subscriptions
+extension overrides the solidus orders api endpoint to accept nested attributes
+for SolidusSubscriptions::LineItems.
+
+## PATCH `/api/orders/:id`
+*Authentication Required* An api or order token must be provided
```
Comment (brendandeere) [inline docs/api/v1/orders.md]: Agreed, I was having trouble deciding how to format it

**Comment by brendandeere** [comment]
@alexblackie fixed updated! Thanks for the proof read :)

**Pull request merged**